codeinterpreterapi/prompts/remove_dl_link.py

```diff
@@ -1,4 +1,5 @@
-from langchain.prompts.chat import ChatPromptTemplate, HumanMessagePromptTemplate
+from langchain.prompts.chat import (ChatPromptTemplate,
```

Comment:
why did you change this?

Reply:
I think this change was caused by black auto-format